docs/manager.md

```diff
@@ -0,0 +1,254 @@
+{"L":"INFO","T":"2020-09-23T11:26:54.519Z","M":"Checking database connectivity...","_trace_id":"xQhkJ0OuR8e6iMDEpM62Hg"}
+```
+
+If there are no errors in the logs, let's spin a Scylla Cluster.
+
+## Cluster registration
+
+When the Scylla Manager is fully up and running, lets create a regular instance of Scylla cluster. 
+
+See [generic tutorial](generic.md) to spawn your cluster.
+
+Once cluster reaches desired node count, cluster status will be updated with ID under which it was registered in Manager.
+
+ ```console
+kubectl -n scylla describe Cluster
+
+[...]
+Status:
+  Manager Id:  d1d532cd-49f2-4c97-9263-25126532803b
+  Racks:
+    us-east-1a:
+      Members:        3
+      Ready Members:  3
+      Version:        4.0.0
+```
+You can use this ID to talk to Scylla Manager using `sctool` CLI installed in Scylla Manager Pod.
+You can also use Cluster name in `namespace/cluster-name` format.
+
+```console
+kubectl -n scylla-manager-system exec -ti scylla-manager-scylla-manager-7bd9f968b9-w25jw -- sctool task list
+
+Cluster: scylla/simple-cluster (d1d532cd-49f2-4c97-9263-25126532803b)
+╭─────────────────────────────────────────────────────────────┬──────────────────────────────────────┬────────────────────────────────┬────────╮
+│ Task                                                        │ Arguments                            │ Next run                       │ Status │
+├─────────────────────────────────────────────────────────────┼──────────────────────────────────────┼────────────────────────────────┼────────┤
+│ healthcheck/400b2723-eec5-422a-b7f3-236a0e10575b            │                                      │ 23 Sep 20 14:28:42 CEST (+15s) │ DONE   │
+│ healthcheck_rest/28169610-a969-4c20-9d11-ab7568b8a1bd       │                                      │ 23 Sep 20 14:29:57 CEST (+1m)  │ NEW    │
+╰─────────────────────────────────────────────────────────────┴──────────────────────────────────────┴────────────────────────────────┴────────╯
+
+```
+
+Scylla Manager by default registers recurring healhcheck tasks for Agent and for each of the enabled frontends (CQL, Alternator).
+
+In this task listing we can see CQL and REST healthchecks.
+
```

Comment:
Is it possible to add the manager to an existing scylla cluster? Might be a good second example.

Reply:
> Is there a particular reason to not use git submodules?

We have a plan to extract scyllaclient and mermaidclient into separate repository, then we will use Go Modules to pull it as a dependency. But since we want to move forward without waiting for it i simply copy pasted it and did necessary modifications in this repo.

Reply:
It is, procedure is the same in both cases. I'll add a note.